framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1602,4 +1642,8 @@ private void handleExpr(AnnotationBuilder builder, String name, Expression expr)
         faexprcache.put(faexpr, res);
         return res;
     }
+
+    public Map<String, TypeElement> getImportedTypes() {
```

Comment:
Why add a public accessor for this field? There are just two invocations, which should simply directly access the field.